### PR TITLE
const evaluatable future compat warning

### PR DESCRIPTION
**Pull request by lcnr**
blocked on https://github.com/rust-lang/rust/pull/74595. Might still need some improvements :shrug: 

r? @nikomatsakis

### CODE REVIEW

**Review by nikomatsakis — APPROVED** [review]

**Comment by nikomatsakis** [comment]
This seems pretty good to me -- cc @rust-lang/lang

File: posts/inside-rust/2020-09-04-Const-Evaluatable.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+---
+layout: post
+title: "Const evaluatable feature compatability"
```
Comment (therealprof) [inline posts/inside-rust/2020-09-04-Const-Evaluatable.md:3]: Evaluatable is a pretty strange word, did you mean evaluable?

Comment (lcnr) [reply posts/inside-rust/2020-09-04-Const-Evaluatable.md:3]: they apparently mean the same thing and we use "evaluatable" inside of the compiler here.

Not sure why this is the case though :thinking:

Comment (therealprof) [reply posts/inside-rust/2020-09-04-Const-Evaluatable.md:3]: According to typical dictionaries (e.g. Merriam-Webster) it's not a real English word.The only place where it seems to be used is Wolfram Research software.

Comment (jyn514) [reply posts/inside-rust/2020-09-04-Const-Evaluatable.md:3]: Maybe this should just say 'evaluation' so it's more clear?

**Comment by lcnr** [comment]
do we still want to publish this post? :laughing:

File: posts/inside-rust/2020-09-04-Const-Evaluatable.md (hunk 1 above)
Comment (lqd) [reply posts/inside-rust/2020-09-04-Const-Evaluatable.md:3]: BTW: 
```suggestion
title: "Const evaluatable feature compatibility"
```

**Pull request closed without merging**